.github/workflows/codestyle.yml

```diff
@@ -7,12 +7,13 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@master
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
           node-version: '12.18.1'
+          cache: 'yarn'
       - name: Install Dependencies
         run: |
-          yarn
+          yarn --frozen-lockfile
```

Comment:
Adding `--frozen-lockfile` is a better practice when installing dependencies on some CI machine.

As per yarn's [documentation](https://classic.yarnpkg.com/en/docs/cli/install/):
> If you need reproducible dependencies, which is usually the case with the continuous integration systems, you should pass --frozen-lockfile flag.